python/kfserving/README.md

```diff
@@ -20,6 +20,8 @@ KFServing supports the following storage providers:
     * Absolute path: `/absolute/path` or `file:///absolute/path`
     * Relative path: `relative/path` or `file://relative/path`
     * For local filesystem, we recommended to use relative path without any prefix.
+* Azure blob Storage with a format: "https://<<StrorageAccountName>>.blob.core.windows.net/<<Path>>"
```

Comment:
nit spacing, also can you use the back tick instead of quotes and {} instead of <<>>?
```suggestion
    * Azure Blob Storage URI: `https://{STORAGE_ACCOUNT_NAME}.blob.core.windows.net/{CONTAINER}/{PATH/ON/CONTAINER}`
```